service/src/main/webapp/DepartmentPage.jsp

```diff
@@ -8,10 +8,10 @@
     <body class="four-column">
```

Comment:
*nitpick*: DepartmentPage.jsp should follow same pattern as other pages and be lowercase *department.jsp*.